pages/[username].js

```diff
@@ -88,7 +88,7 @@ export default function User({ data, BASE_URL }) {
     <>
       <PageHead
         title={data.name}
-        description={data.bio}
+        description={data.bio.replaceAll("*","")}
```

Comment:
we will also need to remove other types of markdown that people might use

Reply:
Oh yeah! I missed that. I'll try to create a function before passing in to `description`

Reply:
It might be easier to do a regex that extracts the text and punctuation

Reply:
Oh, like just letters, spaces and commas?

Reply:
I looked around for this. We can certainly do this with regex. It may not take into account all the complex cases. I used chatgpt to create it for as many cases as we can. We can try this for now and add an external library if this fails